src/frontend/Sider/Behandling/RevurderFra/RevurderFra.tsx

```diff
@@ -60,28 +66,41 @@ export function RevurderFra() {
         }
     }
 
+    const stønadstypeErTilsynBarn = (stønadstype: Stønadstype): boolean => {
+        return stønadstype === Stønadstype.BARNETILSYN;
+    };
+
     return (
         <Container>
-            <VStack gap="2">
-                <DateInputMedLeservisning
-                    label="Revurderes fra"
-                    value={revurderFraDato}
-                    onChange={håndterEndretDato}
-                    erLesevisning={!behandlingErRedigerbar}
-                    feil={valideringsfeil}
-                    size="small"
-                />
-                {visAdvarselOmNullstilling && (
-                    <Alert variant="warning" size="small" style={{ maxWidth: 'fit-content' }}>
-                        Du ønsker nå å revurdere fra en senere tid enn det som tidligere er valgt (
-                        {formaterNullableTilTekstligDato(behandling.revurderFra)}). Hvis du lagrer
-                        denne datoen, blir alle endringene i revurderingen nullstilt.
-                    </Alert>
-                )}
-                {behandlingErRedigerbar && (
-                    <SmallButton onClick={lagreRevurderFraDato}>Lagre og gå videre</SmallButton>
-                )}
-                {feilVedLagring && <Feilmelding>{feilVedLagring}</Feilmelding>}
+            <VStack gap="16">
+                <VStack gap="2">
+                    <DateInputMedLeservisning
+                        label="Revurderes fra"
+                        value={revurderFraDato}
+                        onChange={håndterEndretDato}
+                        erLesevisning={!behandlingErRedigerbar}
+                        feil={valideringsfeil}
+                        size="small"
+                    />
+                    {visAdvarselOmNullstilling && (
+                        <Alert variant="warning" size="small" style={{ maxWidth: 'fit-content' }}>
+                            Du ønsker nå å revurdere fra en senere tid enn det som tidligere er
+                            valgt ({formaterNullableTilTekstligDato(behandling.revurderFra)}). Hvis
+                            du lagrer denne datoen, blir alle endringene i revurderingen nullstilt.
+                        </Alert>
+                    )}
+                    {behandlingErRedigerbar && (
+                        <SmallButton onClick={lagreRevurderFraDato}>Lagre og gå videre</SmallButton>
+                    )}
+                    {feilVedLagring && <Feilmelding>{feilVedLagring}</Feilmelding>}
+                </VStack>
+                {viseVedtakshistorikk &&
+                    stønadstypeErTilsynBarn(behandling.stønadstype) &&
+                    behandling.forrigeBehandlingId && (
+                        <VedtakshistorikkTilsynBarn
+                            forrigeBehandlingId={behandling.forrigeBehandlingId}
+                        />
+                    )}
```

Comment:
For å unngå sjekk på ulike stønadstyper her, hadde det vært enklere å bryte ut hele vedtaksperioder for revurder fra i en egen komponent?

Reply:
Fixed: 574d49959f2881a0ae9f41b5a6abfb2fb57750ed